Command/CreateContentCommand.php

```diff
@@ -35,13 +35,13 @@ protected function configure()
 
     protected function execute( InputInterface $input, OutputInterface $output )
     {
-/** @var $repository \eZ\Publish\API\Repository\Repository */
-$repository = $this->getContainer()->get( 'ezpublish.api.repository' );
-$contentService = $repository->getContentService();
-$locationService = $repository->getLocationService();
-$contentTypeService = $repository->getContentTypeService();
+        /** @var $repository \eZ\Publish\API\Repository\Repository */
+        $repository = $this->getContainer()->get( 'ezpublish.api.repository' );
+        $contentService = $repository->getContentService();
+        $locationService = $repository->getLocationService();
+        $contentTypeService = $repository->getContentTypeService();
 
-        $repository->setCurrentUser( $repository->getUserService()->loadUser( 14 ) );
+        $this->getRepository()->getPermissionResolver()->setCurrentUserReference( $this->getRepository()->getUserService()->loadUser( 14 ) );
```

Comment:
1. You can use `$repository` instead of `$this-getRepository()`.
2. Good chance to fix bad practice relying on hardcoded id. Look at e.g [this command](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Bundle/EzPublishCoreBundle/Command/CopySubtreeCommand.php#L91-L113) to see newest best practice. Ideally this entire block setting user context should be moved to `initialize` as in the mentioned command.